modular_zubbers/code/modules/quirks/code/positive_quirks/empath_override.dm

```diff
@@ -0,0 +1,3 @@
+/datum/quirk/empath
+	name = "Empath"
```

Comment:
You don't need this either. Basically what you are doing is overriding the original value of 6 with 4. The compiler will first compile the original code, then when it compiles our modular changes, it'll see datum path again and duplicate variables with what you set here. (Or add an entirely new variable if you're adding one)